### ISSUE TITLE
Check kube-scheduler restarting in Operator

### ISSUE THREAD

**Issue by safronovD** [dell/csi-baremetal#388]
1. Move logic of kube-scheduler restart waiting from e2e tests to Operator
2. Create flag if CSI is Running or deploy csi-baremetal-node only after restart

**Comment by mishoyama** [comment]
Was implemented under another issue.
